modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java

```diff
@@ -98,12 +121,12 @@ public static boolean deleteDirectory(File directory) {
      * Delete all the files and sub directories which matches given prefix in a given directory.
      *
      * @param directory Directory which contains files to delete.
-     * @param prefix    Prefix for matching files to delete.
+     * @param affix    Prefix for matching files to delete.
```

Comment:
Shall we change the param description according to the affix?

Reply:
Fixed.